### DISCUSSION TITLE
DaVinci Resolve Imports

### DISCUSSION THREAD

**Discussion by WyattBlue** [WyattBlue/auto-editor#504] [3. General]
## FCP11 backend
```
auto-editor example.mp4 --export resolve
```
Issues:
 - Audio tracks > 1 are not linked properly
 - Audio tracks > 1 use an external way because DaVinci Resolve does not appear to support this in FCP11 unlike in Final Cut Pro itself.

## FCP7 backend
Usage:
```
auto-editor example.mp4 --export resolve-fcp7
```
Issues:
 - Audio tracks are always use the "mono" layout.
   - This is fixable. First, right click "1.0" then select "Change Track Type to > Stereo". Then hold down the option key, and select all audio clips, right click, select "Clip Attributes", then go to the "audio" tab. Change "Format" to stereo.
 - Auto-Editor does not create external wav files because you can use the above workaround to point audio clips to the right audio stream. 
   - Change the "source channels" to "channel 3" and "channel 4" to point to the second audio stream if the layout is stereo. 5 and 6 if  their is a third audio stream, and so on.
 - Setting a speed != 1 always causes the clip to start at the beginning
 - Audio tracks > 1 are not linked

**Comment by xemles** [comment]
Any ETA for the audio tracks over 1 to be linked?

**Comment by visualpizza** [comment]
this is the sole thing that keeps me from buying the gui version
